domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputIsEquivalentToRuleClassifierProviderTest.kt

```diff
@@ -156,7 +175,7 @@ class FractionInputIsEquivalentToRuleClassifierProviderTest {
   }
 
   @Test
-  fun testEquality_fraction2Over33Answer_withFaction1Over242Input_bothValuesNotEquivalent() {
+  fun testEquality_fraction33Over2Answer_withFraction242Over1Input_bothValuesNotEquivalent() {
```

Comment:
Nit: can you update all tests here to start with 'testEquivalence' rather than equality? The current names are a bit misleading since the difference is really important.

Reply:
Done